packages/api/src/tx/converter-utils.ts

```diff
@@ -0,0 +1,4 @@
+
+export function AminoDate(d?: Date): string {
+    return d?.toISOString().split('.')[0] + 'Z'
```

Comment:
if d == null then this will return `"undefinedZ"`.

I think we probably want to return empty string, and then ensure that in the converters we do `AminoDate(dateVal) || undefined` ?

This function could probably be changed to sth like:
```suggestion
    return (d ? d.toISOString().split('.')[0] + 'Z' : "")
```